src/components/Error.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect } from 'react';
+import cn from 'classnames';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.